.gitlab-ci.yml

```diff
@@ -841,6 +843,7 @@ test_backend_integration:
       - report_backend_integration_*.html
       - sysstat.log
       - sysstat.svg
+      - /tmp/acceptance.*.gz
```

Comment:
For GitLab to collect any artifacts, they must be in `${CI_PROJECT_DIR}`. If you see above, we use the after_script task to copy anything out of that dir into `.`, which will make it available for the artifact collection.

Reply:
> For GitLab to collect any artifacts, they must be in `${CI_PROJECT_DIR}`. If you see above, we use the `after_script` task to copy anything out of that dir into `.`, which will make it available for the artifact collection.

thanks! how about now?